reference/promise-types/packages.markdown

```diff
@@ -24,6 +24,9 @@ following package managers:
 * ``yum``: YUM package manager and accompanying rpm package manager.
 * ``apt_get``: Apt package manager and accompanying dpkg package manager.
 
+Both ``yum`` and ``apt_get`` package managers require Python to be installed on
```

Comment:
I admit that I haven't tested with both, but I did use the recommended syntax which is supposed to work on both. Until we know of any problems I'll leave it unspecified, and if we find any, I think we should fix the code instead.